Resources/public/css/theme/views/loginform.css

```diff
@@ -24,8 +24,9 @@
 }
 
 .ez-view-loginformview .ez-loginform-title {
-    font-size: 0.9rem;
+    font-size: 0.95rem;
     color: #444;
+    font-family: sans-serif;
```

Comment:
it's a good practice to put: `font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;` because, Helvetica is not available on Windows: http://snook.ca/archives/html_and_css/windows-subs-helvetica-arial